nerdlets/root/function-picker.js

```diff
@@ -17,20 +21,17 @@ export default class FunctionPicker extends React.PureComponent {
 
     if(!attribute || isCount) return <div/>
 
-    return <div className="function-picker-radio-container">
-      {FUNCTIONS.map(f => {
-        const checked = fn == f.fn
-        return (
-          <Radio
-            key={f.fn}
-            onClick={() => setFunction(f.fn)}
-            checked={checked}
-            label={f.label}
-            className="function-picker-radio"
-            />
-        )
-      })}
-
-    </div>
+    const options = FUNCTIONS
+    return (
+      <div className="react-select-input-group">
+        <label>Function</label>
+        <Select
+          options={options}
+          value={{ value: fn, label: 'label' }} //TODO: get label to update based on state
+          onChange={s => setFunction(s.fn)}
```

Comment:
The last change is updating the argument that is being passed into `setFunction()`. Once you've update the `FUNCTIONS` const, then the this argument being passed into `setFunction()` should be `s.value` rather than `s.fn`